modules/check_in/app/services/map/service.rb

```diff
@@ -50,5 +50,14 @@ def fetch_or_generate_token
 
       token_response[:access_token]
     end
+
+    def get_appointments(token:, patient_identifier:, time_period:)
+      resp = if token.present?
+               client.get_appointments(token:, patient_identifier:, time_period:)
+             else
+               Faraday::Response.new(body: { message: 'Unauthorized' }, status: 401)
+             end
+      render json: VAOS::V2::AppointmentsCheckInSerializer.new(resp[:data], meta: resp[:meta])
```

Comment:
Lee, `render` method is part of [ActionController::Base class](https://apidock.com/rails/ActionController/Base/render).  Advisable to use in the controller in layer and not in the service layer.

Reply:
MAP appointments endpoint sends `data` in `200` successful responses whereas it will be missing other failure scenarios.  

Could you please see how to serialize the response for all the scenarios?  Also, please clarify what is the purpose of `resp[:meta] `

Reply:
Updated, remove meta, added new response object, shortened old code :)